tests/e2e_suite.py

```diff
@@ -22,6 +22,19 @@ class TestStatus(Enum):
     skipped = 3
 
 
+def mem_stats(network):
+    mem = {}
+    for node in network.get_joined_nodes():
+        try:
+            primary, _ = network.find_primary()
+            with primary.client() as c:
```

Comment:
Don't think we should always be contacting the primary?

```suggestion
            with node.client() as c:
```

Reply:
Indeed no, that got leftover from before.